gadopt/energy_solver.py

```diff
@@ -139,9 +139,8 @@ def setup_solver(self):
                                    solver_parameters=self.solver_parameters)
         self._solver_setup = True
 
-    def solve(self):
+    def solve(self, t=0, update_forcings=None):
```

Comment:
Why is this t=0 option there? Just something Sia also mentioned, so important to have documented.

Reply:
It was always there, but not used. `t` is given to `update_forcings` and is needed for the Trim case, for example.